force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -378,6 +400,13 @@ public with sharing class ClassesObjectsMethods {
      * @return The Person object of the older person.
      */
     public static Person getOlderPerson(Person person1, Person person2) {
-        return null; // Replace null with the variable you used to store the result
+        if (person1.getAge() > person2.getAge()) {
```

Comment:
This logic can be simplified to return person1 when person1's age is greater than or equal to person2's. Otherwise, return person2